src/main/java/me/glasscrab/plentiful_harvest/Commands/GiveCropCommand.java

```diff
@@ -9,24 +11,35 @@
 import org.bukkit.entity.Player;
 import org.bukkit.inventory.ItemStack;
 
-import java.util.ArrayList;
-import java.util.List;
+import me.glasscrab.plentiful_harvest.Manager;
+import me.glasscrab.plentiful_harvest.PlentifulHarvest;
+import net.kyori.adventure.audience.Audience;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.minimessage.MiniMessage;
 
 public class GiveCropCommand implements CommandExecutor {
     @Override
     public boolean onCommand(CommandSender sender, Command command, String s, String[] args) {
+
+        var miniMessage = MiniMessage.miniMessage();
+        Audience audience = PlentifulHarvest.INSTANCE.audiences.sender(sender);
```

Comment:
Instead of using Audience, just use the CommandSender, it still has the sendMessage function